src/redux/news/news.sagas.js

```diff
@@ -104,10 +89,7 @@ function* handleNewsUpdate({ payload }) {
   const { id, newArticle } = payload;
   try {
     yield put(setNewsLoading(true));
-    const result = yield call(updateArticle, id, newArticle);
-    if (result.message) {
-      yield call(handleCustomNewsError, result.message[errorsLanguage].value);
-    }
+    yield call(updateArticle, id, newArticle);
```

Comment:
Why do we fetch again all the news after the updating of the article? It is a useless HTTP call. Maybe, it is much better to just take the result of mutation and override old article with the new data. It will reduce the quantity of HTTP requests: 2 requests => 1 request